docs/source/event-descriptors.rst

```diff
@@ -0,0 +1,210 @@
+Event Descriptors
+=================
+
+In the section on :doc:`documents`, we gave an overview of the four kinds of
+document. We presented an example Run Start, Event, and Run Stop, but we
```

Comment:
This second sentence is no longer accurate. We do present an Event Descriptor example. We just down break it down and discuss it in detail.

Reply:
check